notebooks/01-lightcurves/01-Lightcurves.ipynb

```diff
@@ -56,9 +56,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #We can get an exact id from our table we quieried above
</pre><p> </p><p>queried</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>fixed!</p>